### ISSUE TITLE
can you implement with just 1 for? iterate only one time

### ISSUE THREAD

**Issue by radumarias** [radumarias/Pavel_Ruban_interview#1]
https://github.com/radumarias/Pavel_Ruban_interview/blob/3ef9d7ef56f822e551715fd74a27b2ea7c6c4da7/problem_2/src/main/java/com/test/problem2/service/impl/PangramVerifierService.java#L27

**Comment by radumarias** [comment]
also maybe you can try without the frequency array?

**Comment by PavelRuban** [comment]
There is only one iteration through input string. I think I can avoid iteration on alphabet flags (or frequency array) using hashset of chars and put letters there. finally check the size. but there is one problem - have to transform whole string to upper or lower case first. this transformation performs one more iteration on input string, plus it creates new string, which finally doesn't seems more effective for me..

**Comment by radumarias** [comment]
agree, I didn't refer to that solution, there is one even optimal :)
